motionplan/armplanning/real_test.go

```diff
@@ -0,0 +1,47 @@
+package armplanning
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"go.viam.com/test"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/referenceframe"
+)
+
+func readRequestFromFile(f string) (*PlanRequest, error) {
+	content, err := os.ReadFile(f)
+	if err != nil {
+		return nil, err
+	}
+
+	req := &PlanRequest{}
+
+	err = json.Unmarshal(content, req)
+	if err != nil {
+		return nil, err
+	}
+
+	return req, nil
+}
+
+func TestOrb1(t *testing.T) {
+	logger := logging.NewTestLogger(t)
+
+	req, err := readRequestFromFile("data/orb-plan1.json")
+	test.That(t, err, test.ShouldBeNil)
+
+	for i := 0; i < 100; i++ {
+		req.PlannerOptions.RandomSeed = i
+		plan, err := PlanMotion(context.Background(), logger, req)
+		test.That(t, err, test.ShouldBeNil)
+
+		a := plan.Trajectory()[0]["sanding-ur5"]
+		b := plan.Trajectory()[1]["sanding-ur5"]
+
+		test.That(t, referenceframe.InputsL2Distance(a, b), test.ShouldBeLessThan, .005)
```

Comment:
What exactly is this testing that the first and second joint positions of each plan are not too dissimilar from each other?

If so, why is that what we care about in this test?

Reply:
because we don't want the "pirouette" issue